### DISCUSSION TITLE
I NEED MORE ULTRAKILL SONGS

### DISCUSSION THREAD

**Discussion by Emp1r1c** [WebOsu-2/webosu-2.github.io#16] [Ideas]
I am having a lack of ultrakill songs and I need them in I know it another version of webosu had some but I cant get onto that version anymore so I cant play them but I just highly  recommend for someone to make them anyways that's all

**Comment by Mostlime12195** [comment]
I love ultrakill too. The issue here is that the API we use to get all of our beatmaps (Mino) is missing a very large chunk of beatmaps. So, for that reason, I will work towards switching our API from Mino back to SayoBot (the original one). I'll let you know when the move is done.

**Reply by Mostlime12195** [reply]
BOOM! Done. Within a couple hours, I reverted all the Mino code back to SayoBot. Enjoy all your new beatmaps! I love Tenebre Rosso Sangue, I'll play it so much.

@Emp1r1c just pinging you so you know about the change.

**Reply by Emp1r1c** [reply]
THANK YOU I JUST SAW THIS
